src/index.ts

```diff
@@ -534,4 +534,45 @@ export class Supertab {
       siteLogoUrl,
     };
   }
+
+  async getCurrencyDetails(isoCode: string) {
+    const experiencesConfig = await this.#getClientExperiencesConfig();
+
+    if (!isoCode) {
+      isoCode = experiencesConfig.suggestedCurrency || DEFAULT_CURRENCY;
+    }
+
+    const currency = experiencesConfig.currencies.find(
+      (currency) => currency.isoCode === isoCode,
+    );
+
+    if (!currency) {
+      return null;
+    }
+
+    const formatTabLimit = (amount: number) =>
+      formatPrice({
+        amount,
+        currency: currency.isoCode,
+        baseUnit: currency.baseUnit,
+        localeCode: this.language,
+        showZeroFractionDigits: false,
+        showSymbol: currency.isoCode !== "CHF",
+      });
+
+    return {
+      isoCode: currency.isoCode,
+      name: currency.name,
+      symbol: currency.symbol,
+      baseUnit: currency.baseUnit,
+      firstTabLimit: {
+        amount: currency.firstTabLimit,
+        text: formatTabLimit(currency.firstTabLimit ?? 100),
```

Comment:
extracting `100` (and `500` bellow) to the `const` with a meaningful name could help readability here

Reply:
Thank you for the suggestion. Addressed in https://github.com/laterpay/supertab-browser/pull/147/commits/c8fde968984ca2f4799978a8c22d6f7fa891ebed.